### ISSUE TITLE
Examine prerequisites when a student visits an exercise

### ISSUE THREAD

**Issue by bcipolli** [learningequality/ka-lite#1150]
the current KA topic tree has a field "prerequisites", that lists all of the exercise_ids of an exercise that presumably should be done before a student proceeds to the current one.

We can tell the user this information, to help them "go in order" or to help show progress along a chain.

**Comment by rtibbles** [comment]
Superseded by #3368
